### ISSUE TITLE
Div tags in description

### ISSUE THREAD

**Issue by AdrianTwo** [AdrianTwo/ud_definition_grabber#1]
When making a query like "udd_grabber lmfao" the result is following:

```
lmfao
------------------------------------------------------------------------------------------------------------------------
Laughing my fucking ass off</div><div class="example">Someone told me Michelle deserved to win Pop Idol, and I was LMFAO
------------------------------------------------------------------------------------------------------------------------
Someone told me Michelle deserved to win Pop Idol, and I was LMFAO
------------------------------------------------------------------------------------------------------------------------
by Hmm. December 23, 2003

```
This is incorrect because of the additional tags and text in the description area.

**Comment by AdrianTwo** [comment]
This happens when the description consists of only one autolink.

**Comment by AdrianTwo** [comment]
Solved issue in commit f828fe7033824bb76c317de6780c26a4b7c15bec
